Most major airlines allow passengers to carry two pieces of luggage (of a certain maximum size) onto
the plane. However, their studies show that the more carry-on baggage passengers have, the longer it
takes to unload and load passengers. One regional airline is considering changing its policy to allow only
one carry-on per passenger. Before doing so, it decided to collect some data. Specifically, a random
sample of 1,000 passengers was selected. The passengers were observed, and the number of bags
carried on the plane was noted. Out of the 1,000 passengers, 345 had more than one bag.

a) Based on this sample, develop and interpret a 95% confidence interval estimate for the
proportion of the traveling population that would have been impacted had the one-bag limit
been in effect. Discuss your result.

In [34]:
n = 1000
a = 345

affected = a / n #compute point estimate

In [35]:
print(affected)

0.345


CONFIDENCE INTERVAL ESTIMATE FOR POP. PROPORTION

In [36]:
import scipy.stats as st 
import math as m
import numpy as np

In [37]:
z = st.norm.ppf((1-0.95) / 2) # get critical value
print (z)

-1.95996398454


In [38]:
pp = affected * (1 - affected) / p  # calculate based on formula the value before being square root

In [39]:
#Calculate the upper tail

affected + z * m.sqrt(pp)

0.3155369102603553

In [40]:
#Calculate the lower tail

affected - z * m.sqrt(pp)

0.37446308973964465

It will be 37.4% to 31.6% traveling population to be impact.

b) The domestic version of Boeing’s 747 has a capacity for 568 passengers. Determine an interval
estimate of the number of passengers that you would expect to carry more than one piece of
luggage on the plane. Assume the plane is at its passenger capacity.

In [41]:
#new population proportion

nn = 568
new_pp = affected * (1 - affected) / nn  # calculate based on formula the value before being square root
first = affected + z * m.sqrt(new_pp)
second = affected - z * m.sqrt(new_pp)

print (first)
print (second)

0.305906539013
0.384093460987


It will be 30.6% to 38.4% passengers travelling with more than 1 bag